### ISSUE TITLE
What is the plan for this NuGet/repository?

### ISSUE THREAD

**Issue by GintasS** [ardalis/Ardalis.Extensions#6]
First of all, thank you for this repository and NuGet package. It can really be useful.

But I was wondering, what is the plan for this NuGet package? Because I can see people adding some extension methods.
Some questions:

- Some of these methods that people are adding, are not full-proof, we need to make sure that they're are resilient and does the job 100%.
- Some of these methods are lacking proper documentation - XML structure. They should have proper XML comments, with remarks, external links, potential exceptions, return values and so on.
- Tests should be too more rigorous, testing not one, but many possible inputs.
- Changelog could be added to the repository too.
- StyleAnalizers or something to unify code style would be nice too.

**TL:DR** What are the plans for this repository? Is this one time, initial contribution or there are any future plans?

**Comment by ardalis** [comment]
I agree 100% that the methods should be well-documented, well-tested, useful, and consistently styled. That's the goal. Contributions are welcome and I will eventually make this a 1.0 release when I think it's good enough for public consumption.
